src/issue-listing.jsx

```diff
@@ -0,0 +1,525 @@
+import React, { Component } from 'react';
+import styled from 'styled-components/macro';
+import Octokit from '@octokit/rest';
+import Moment from 'react-moment';
+
+import theme from './theme';
+import mediaQueries from './media-queries';
+
+import { Container } from './components/container';
+import { Row } from './components/row';
+import { LoadingSpinner } from './components/loading-spinner';
+import { FormInput } from './components/form-input';
+import { Image } from './components/image';
+import { Button } from './components/button';
+import SmallArrow from './components/small-arrow';
+
+const IssueListingContainer = styled(Container)`
+	&.no-repo-selected {
+		display: none;
+
+		@media (min-width: ${mediaQueries.min.medium}) {
+			display: block;
+		}
+	}
+`;
+IssueListingContainer.displayName = 'IssueListingContainer';
+
+const NoIssuesMessage = styled.p`
+	padding-left: ${theme.gutter};
```

Comment:
Indentation looks a bit off in this file! Prettier will handle it for you, or even `eslint fix` if you've got eslint already!